src/main/java/core/basesyntax/service/Transfer.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface Transfer {
+
+    void generateInfo(List<FruitTransaction> info);

```

Comment:
Think more about naming here. `generateInfo` - if the method generates something it can not be **void**.
Why `List<FruitTransaction>` you called  **info**?
See my suggestion :)
```suggestion
public interface TransactionExecutor {
    void execute(List<FruitTransaction> transactions);
```